pkgs/tools/backup/iceshelf/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, pythonPackages, git, awscli, python }:
+
+pythonPackages.buildPythonApplication rec {
+  pname = "iceshelf";
+  version = "unstable-2019-07-03";
+
+  format = "other";
+
+	src = fetchFromGitHub {
+		owner = "mrworf";
+		repo = pname;
+		rev = "26768dde3fc54fa412e523eb8f8552e866b4853b";
+		sha256 = "08rcbd14vn7312rmk2hyvdzvhibri31c4r5lzdrwb1n1y9q761qm";
+  };
+
+  propagatedBuildInputs = [ git awscli ] ++ (with pythonPackages; [ python-gnupg ]);
```

code style wise I'd prefer

```suggestion
  propagatedBuildInputs = [ 
    awscli 
    git
    python3.pkgs.python-gnupg
  ];
```